internal-enrichment/shodan-internetdb/src/shodan_internetdb/config.py

```diff
@@ -30,6 +33,16 @@ def _load_config() -> dict[str, Any]:
             else {}
         )
 
+    def _check_configuration(self) -> None:
+        """
+        Run configuration additional checks not handled by get_config_variables function.
+        """
+        if self.shodan_max_tlp not in TLP_MARKING_OPTIONS:
+            raise ValueError(
+                "Incorrect value for configuration parameter 'max_tlp'. "
+                "Permitted values are: 'TLP:WHITE', 'TLP:GREEN', 'TLP:AMBER', 'TLP:RED' or 'TLP:CLEAR'"
```

Comment:
you can do it this way if you want :
```suggestion
                f"Permitted values are: {', '.join(TLP_MARKING_OPTIONS)}"
```